src/include/miopen/solver.hpp

```diff
@@ -2305,6 +2295,58 @@ struct GemmBwdRest : GemmBwdBase
     ConvSolution GetSolution(const ExecutionContext&, const conv::ProblemDescription&) const;
 };
 
+struct GemmWrwBase : SolverBase<ConvolutionContext>
+{
+    bool IsApplicable(const ExecutionContext&, const conv::ProblemDescription&) const;
+    bool IsDynamic() const { return true; }
+    float GetWti(const ConvolutionContext& ctx) const { return GetWti(ctx, ctx.conv_problem); }
+    float GetWti(const ExecutionContext& context, const conv::ProblemDescription& problem) const;
```

Comment:
Make it non-public.

Reply:
Why? We are going to remove old ones and use only these methods after we update all the solvers to the new problem config.

Reply:
Okay